recipes/cereal/all/conanfile.py

```diff
@@ -32,11 +35,34 @@ def package(self):
         cmake.configure()
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "share"))
+        self._create_module_official_cmake_targets(
+            os.path.join(self.package_folder, self._module_folder, self._module_file),
+            [{"official": "cereal", "namespaced": "cereal::cereal"}]
+        )
 
-    def package_id(self):
-        self.info.header_only()
+    @staticmethod
+    def _create_module_official_cmake_targets(module_file, targets):
```

Comment:
Should this be added as a tool to the client?

Reply:
yes, it would be nice

Reply:
@SpaceIm It looks a Conan case not a package feature. It would be great opening an issue to talk about, so we can internalize it.

Reply:
Yes, there is a suggestion of @madebr https://github.com/conan-io/conan/pull/8338#discussion_r557653422

Let's open an issue in https://github.com/conan-io/conan

Reply:
Done: https://github.com/conan-io/conan/issues/8360